app/src/main/res/layout/button_download.xml

```diff
@@ -17,13 +17,16 @@
 
 <ImageButton
     xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/AppTheme.Secondary.ImageButton"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Why did you change the button theme?

Reply:
That's what I already disliked about the old impl., that the download wasn't really visualized like a button. This way, I find it quite easier for the user.

Reply:
Please do not change the style individual for each item. Use styles instead. Otherwise it gets a mess again. I tried to clean this up with PR #217.